src/components/new-request/search.vue

```diff
@@ -144,11 +150,22 @@
 
         <template v-if="selectedCompanyType === CompanyType.NAMED_COMPANY">
           <!-- Xpro/Federal bullets -->
-          <v-col v-if="getIsXproFlow && isFederal" cols="12" md="8">
+          <v-col v-if="getIsXproFlow && isFederal" cols="12" :md="isSelectedXproAndRestorable ? 10 : 8">
             <ul class="bullet-points">
               <li>Federally incorporated businesses do not need a Name Request.</li>
-              <li>You may register your extraprovincial business immediately using its existing name
-                at Corporate Online.</li>
+              <li v-if="!isSelectedXproAndRestorable">
+                You may register your extraprovincial business immediately using its existing name
+                at Corporate Online.
+              </li>
+              <li v-else>
+                To reinstate your business, complete
+                <a :href="fullReinstatementFormLink">
```

Comment:
If a selected restorable business that is also an XPRO is selected, we show this text and form links to download as per the UXPin.